app/src/main/java/com/bagi/flowkotlin/activity/covid19/Covid19ActivityViewModel.kt

```diff
@@ -10,25 +10,29 @@ import java.text.SimpleDateFormat
 import java.util.*
 
 class Covid19ActivityViewModel : ViewModel() {
-    private val countryDetailsMLD = MutableLiveData<MutableList<Triple<Int, Int?, Int?>>>()
-    private val progressDataMLD = MutableLiveData<Boolean>()
-    private val updatedMLd = MutableLiveData<String>()
+    private val _countryDetailsMLD = MutableLiveData<MutableList<Triple<Int, Int?, Int?>>>()
+    private val _progressDataMLD = MutableLiveData<Boolean>()
+    private val _updatedMLd = MutableLiveData<String>()
 
-    fun getCountryDetailsMLD(): LiveData<MutableList<Triple<Int, Int?, Int?>>> = countryDetailsMLD
-    fun getProgressDataMLD(): LiveData<Boolean> = progressDataMLD
-    fun getUpdatedMLd(): LiveData<String> = updatedMLd
+    fun getCountryDetailsMLD(): LiveData<MutableList<Triple<Int, Int?, Int?>>> = _countryDetailsMLD
 
+    fun getProgressDataMLD(): LiveData<Boolean> = _progressDataMLD
+    fun getUpdatedMLd(): LiveData<String> = _updatedMLd
```

Comment:
Why you do that?
why you change the name parameter with underscore prefix?

Reply:
Actually, in this particular example we don't need the underscore, because we don't have conflicts between declarations. if you go to my github example you will see that i used variables instead of functions. i guess i'm just used to it and continued doing that without paying attention...

Reply:
💯